### DISCUSSION TITLE
API

### DISCUSSION THREAD

**Discussion by ExXxtr3me** [ZoeyVid/NPMplus#597] [Q&A] [answered]
Hi, i just switched from NPM to NPMplus and evrything seems to be working fine with my proxies but i can't get the api with the Homepage Dashboard (https://gethomepage.dev/latest/widgets/services/nginx-proxy-manager/ ) to work. Is there something else i have to change?

**Comment by Zoey2936** [comment]
do you use https?

**Comment by ExXxtr3me** [comment]
Yes, i tried it with both.

**Comment by ExXxtr3me** [comment]
I'm seeing this in the NPMplus logs:
![image](https://github.com/ZoeyVid/NPMplus/assets/6366829/37529f6e-ebe0-4dd8-85fb-af4cbd429455)

**Comment by Zoey2936** [comment]
Can you try to resign in in homepage?

**Comment by ExXxtr3me** [comment]
Sorry but what do you mean with "resign" in Homepage?

**Comment by ExXxtr3me** [comment, marked as answer]
Ok it's fixed, i just deleted the widget, restarted Homepage and added it again. 👍
